Support Angular 20

> npm error code ERESOLVE
npm error ERESOLVE unable to resolve dependency tree
npm error
npm error While resolving: Keira3@3.9.2-dev
npm error Found: @angular/core@20.1.4
npm error node_modules/@angular/core
npm error   dev @angular/core@"20.1.4" from the root project
npm error   peer @angular/core@"20.1.4" from @angular/common@20.1.4
npm error   node_modules/@angular/common
npm error     dev @angular/common@"20.1.4" from the root project
npm error     peer overridden @angular/common@"20.1.4" (was "^18.0.0 || ^19.0.0") from ngx-select-ex@19.0.5
npm error     node_modules/ngx-select-ex
npm error       ngx-select-ex@"19.0.5" from the root project
npm error
npm error Could not resolve dependency:
npm error peer @angular/core@"^18.0.0 || ^19.0.0" from ngx-select-ex@19.0.5
npm error node_modules/ngx-select-ex
npm error   ngx-select-ex@"19.0.5" from the root project

--- a/projects/ngx-select-ex/package.json
+++ b/projects/ngx-select-ex/package.json
@@ -3,8 +3,8 @@
   "version": "19.0.5",
   "license": "MIT",
   "peerDependencies": {
-    "@angular/common": "^18.0.0 || ^19.0.0",
-    "@angular/core": "^18.0.0 || ^19.0.0",
+    "@angular/common": ">=18.0.0",
+    "@angular/core": ">=18.0.0",
     "escape-string-regexp": "^5.0.0",
     "lodash.isequal": "^4.5.0"
   },

